src/System.Private.ServiceModel/tools/test/HelixLabRun/launchWCFOpenBuilds.cmd

```diff
@@ -0,0 +1,41 @@
+REM 
+REM This script sync to the latest WCF source, build and submit Helix lab runs.
+REM Arguments:
+REM %1: Lab scripts location
+REM %2: WCF local git source location
+
+set BridgeHost=
+cd %2
+REM ensure bridge is stopped
+call stopbridge.cmd
+REM remove bin directory to ensure a rebuild
+del /s /q bin
+del *.rsp
+
+REM sync to the latest
+git checkout master
+git pull
+
```

Comment:
For a script like this, I recommend checking ERRORLEVEL after every single operation.  If any step fails (like stopping the Bridge or the git pull, the script should report that and stop.